[Mellanox] Add kernel patches with hw-mgmt V.7.0010.1300

keboliu opened the pull request:
Add new kernel patches for mlxsw driver:

-  Move PSU EEPROM configuration from kerenl to user space for SPC2 / SPC3 system 
   official  commit ID  c071afcea6ecf24a3c119f25ce9f71ffd55b5dc2 https://github.com/torvalds/linux/commit/c071afcea6ecf24a3c119f25ce9f71ffd55b5dc2
-  mlxsw: core: thermal: Enable thermal zones by default
    these codes are already in upstream, previously when backporting to SONiC this was set to disable, here we just restore it back, no new code added here comparing to the upstream.

lguohan commented on patch/0015-mlxsw-core-thermal-Enable-thermal-zones-by-default.patch:
```diff
@@ -0,0 +1,48 @@
+Subject: [backport 4.19 3/3] mlxsw: core: thermal: Enable thermal zones by
+ default
+
+Change default setting for the all thermal zones from "disabled" to
```
what is the upstream commit id?

keboliu replied:
Hi Guohan, actually these codes are already in upstream, previously when backporting to SONiC this was set to disable because of the thermal control not available,  here we just restore it back, so no new code added here.

liat-grozovik replied:
@lguohan is it ok to be merged?
by accident i merged the dependent one before this one was taken properly to buildimage.

lguohan replied:
what is the upstream commit id? can you put it into the patch message?

lguohan reviewed (changes requested):
please put upstream commit id in the patch message.

liat-grozovik commented:
@keboliu please split this PR to 2: 1 with all upstream commits and 1 with the non upstream code which is for debug only.
once this is provided we can move forward with the merge.

liat-grozovik commented:
@keboliu please refer to the last comment. We need the upstream commit id in the patch message. once all is clear we can move forarward.

keboliu commented:
@lguohan I removed the patch which not yet merged to linux kernel community yet(which is in next next), please check again.

keboliu commented:
> @lguohan I removed the patch which not yet merged to linux kernel community yet(which is in next next), please check again.

@lguohan the first patch has been already merged to linux upstream, the official commit ID is: 9306c4761912961dc228c32b0de99571c3932928, I also updated the PR description.

lguohan commented:
i cannot find the 9306c4761912961dc228c32b0de99571c3932928 in this repo. https://github.com/torvalds/linux

keboliu commented:
> i cannot find the 9306c4761912961dc228c32b0de99571c3932928 in this repo. https://github.com/torvalds/linux

sorry, typo, check this: c071afcea6ecf24a3c119f25ce9f71ffd55b5dc2    https://github.com/torvalds/linux/commit/c071afcea6ecf24a3c119f25ce9f71ffd55b5dc2

liat-grozovik commented:
@lguohan anything else missing?
